dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts

```diff
@@ -81,3 +86,34 @@ export const MultiSelectFormProps = Object.assign(deepCopyObject(FormFieldPropsW
     required: false,
   },
 }) as Readonly<ComponentPropsOptions>;
+
+export const DataPointFormFieldProps = Object.assign(deepCopyObject(YesNoFormFieldProps as ObjectType), {
+  dataTest: {
+    type: String,
+  },
+  placeholder: {
+    type: String,
+  },
+  evidenceDesired: {
+    type: Boolean,
+  },
+}) as Readonly<ComponentPropsOptions>;
```

Comment:
This is not needed, since it is already inherited by the `YesNoFormFieldProps"

Reply:
Has this been fixed.. or is outdated?

Reply:
Closing since outdated (and also only one occurrence of "evidenceDesired" in the file currently).